feeds/wlan-ap/opensync/src/platform/openwrt/rootfs/common/etc/init.d/opensync

```diff
@@ -47,5 +47,5 @@ stop_service() {
     echo "Removing certificates"
     rm -rf ${CERTS_DEST_PATH}
     echo "Killing managers"
-    killall dm wm sm cm nm lm qm cmdm rrm um uccm
+    killall -s SIGKILL dm wm sm cm nm lm qm cmdm rrm um uccm
```

Comment:
Unrelated.  Should be a separate commit.

Reply:
Done.  This change is in a separate PR.